WebApplication/MigrationJob.cs

```diff
@@ -0,0 +1,38 @@
+/////////////////////////////////////////////////////////////////////
+// Copyright (c) Autodesk, Inc. All rights reserved
+// Written by Forge Design Automation team for Inventor
+//
+// Permission to use, copy, modify, and distribute this software in
+// object code form for any purpose and without fee is hereby granted,
+// provided that the above copyright notice appears in all copies and
+// that both that copyright notice and the limited warranty and
+// restricted rights notice below appear in all supporting
+// documentation.
+//
+// AUTODESK PROVIDES THIS PROGRAM "AS IS" AND WITH ALL FAULTS.
+// AUTODESK SPECIFICALLY DISCLAIMS ANY IMPLIED WARRANTY OF
+// MERCHANTABILITY OR FITNESS FOR A PARTICULAR USE.  AUTODESK, INC.
+// DOES NOT WARRANT THAT THE OPERATION OF THE PROGRAM WILL BE
+// UNINTERRUPTED OR ERROR FREE.
+/////////////////////////////////////////////////////////////////////
+
+using WebApplication.Definitions;
+
+namespace MigrationApp
+{
+   public class MigrationJob
+   {
+      public enum JobType {CopyAndAdopt};
```

Comment:
Where do you plan to use this information?

Reply:
next step (I am working on right now) is to delete projects, which were deleted by the user after migration started. For that, there will be new JobType. Later on we will be migrating also parameter configurations.